Sources/OutlineView/OutlineViewDragAndDrop.swift

```diff
@@ -0,0 +1,139 @@
+import Foundation
+import AppKit
+
+/// A protocol for use with `OutlineView`, implemented by an object (most likely
+/// the data source) to handle receiving items that are dragged into the `OutlineView`.
+@available(macOS 10.15, *)
+public protocol DropReceiver {
+    associatedtype DataElement: Identifiable
+    
+    /// Given a dragged `NSPasteboardItem`, attempts to read the data in the item
+    /// into a usable object for the `OutlineView`'s data source.
+    ///
+    /// - Parameter item: The Pasteboard item that is being dragged into the OutlineView
+    ///
+    /// - Returns: nil if the Pasteboard item is not readable by this receiver, or
+    ///   a tuple with the decoded item and its associated PasteboardType, which must
+    ///   be included in `acceptedTypes`
```

Comment:
```suggestion
    /// - Returns: `nil` if the pasteboard item is not readable by this receiver, or
    ///   a tuple with the decoded item and its associated PasteboardType, which must
    ///   be included in `acceptedTypes`.
```

Reply:
Got it https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3